cmd/web.go

```diff
@@ -41,7 +41,7 @@ and it takes care of all the other things for you`,
 		},
 		cli.StringFlag{
 			Name:  "pid, P",
-			Value: "/var/run/gitea.pid",
```

Comment:
With a default setting here `ctx.IsSet("pid")` below will always be set to true

Reply:
That doesn't appear to be the case...